base/gmp.jl

```diff
@@ -69,6 +69,93 @@ function __init__()
     end
 end
 
+
+module mpz
```

Comment:
Yes I'm sensible to that, but I made this choice to be as close as possible to the original function names, of the form "mpz_add", which translates here into "mpz.add". An alternative could be to not have any submodule and to name the julia functions as in GMP, with the "mpz_" prefix. I prefer the module with lower case, and I thought that an exception to the style can be made as it's an internal sub-sub-module of Base. But will change if you or someone insists.